frontend/src/App.js

```diff
@@ -22,94 +21,85 @@ function App() {
       return null;
     }
     let type = listType(title);
-  
-    setList({title:title, data:data, html:data.map(entry => {
-      if(type == "Shows"){
-          return(
+    setList({
+      title: title, data: data, html: data.map(entry => {
+        if (type == "Shows") {
+          return (
             <div>
-              <Show chooseShow={chooseShow} key={entry} token={token} list={Playlists} show={entry} isMiddle={MiddleShow} all_platforms={all_platforms}/>
+              <Show chooseShow={chooseShow} key={entry} token={token} list={Playlists} show={entry} isMiddle={MiddleShow} all_platforms={all_platforms} />
             </div>
           )
-      } else if(type == "Watchlist"){
-        return(
-          <div>
-            <Playlist chooseWatchlist={chooseWatchlist} watchlist={entry}/>
-          </div>
-        )
-      } 
-    })})
+        } else if (type == "Watchlist") {
+          return (
+            <div>
+              <Watchlist chooseWatchlist={chooseWatchlist} watchlist={entry} />
+            </div>
+          )
+        }
+      })
+    })
   }
 
   useEffect(() => {
 
-    axios.get("http://localhost:3001/platforms").then(response=>{ 
+    axios.get("http://localhost:3001/platforms").then(response => {
       setAllPlatforms(response.data);
     })
-    .catch((error) => {
-      console.log(error);
-    });
+      .catch((error) => {
+        console.log(error);
+      });
 
     const ls = JSON.parse(localStorage.getItem(LS_KEY))
-    if(ls) { //signed in
+    if (ls) { //signed in
       signin(ls.token, ls.username)
-      
-      /* setPlaylists(
-        //axios call to get multiple playlists
-        //this is just an example of one
-        <Playlist watchlist="first one"/>
-      ) */
-      
-    }else{ //not signed in
-      axios.post("http://localhost:3001/shows/recentlyadded").then(response=>{
+
+    } else { //not signed in
+      axios.post("http://localhost:3001/shows/recentlyadded").then(response => {
         generateHTML(setSecondaryList, SecondaryList, "Recently Added", response.data)
       })
-      .catch((error) => {
-        console.log(error);
-      });
+        .catch((error) => {
+          console.log(error);
+        });
     }
   }, []);
 
-  useEffect(()=>{
+  useEffect(() => {
     //setCurrentPage(1);
-    if(all_platforms){
+    if (all_platforms) {
       axios.post("http://localhost:3001/shows",
-      {
-        search_str: searchString
-      }).then(response=>{
-        //setCount(response.data.count);
-        generateHTML(setPrimaryList, PrimaryList, `Results for '${searchString}'`, response.data.data);
-      })
-      .catch((error) => {
-        console.log(error);
-      })
+        {
+          search_str: searchString
+        }).then(response => {
+          generateHTML(setPrimaryList, PrimaryList, (searchString == '') ? 'Top Anime' : `Results for '${searchString}'`, response.data.data);
```

Comment:
everything else is just prettifying/ indentation... this is what I actually changed (lines 38 & 98)